src/libstore/download.cc

```diff
@@ -285,8 +285,12 @@ struct CurlDownloader : public Downloader
             }
 
             if (request.verifyTLS) {
-                if (settings.caFile != "")
+                debug("verify TLS enabled; configured CA file = '%s'", settings.caFile);
```

Comment:
the debug line is redundant when caFile == "", you could move it inside of the if condition

Reply:
In that case I would tend to use a similar prefix on both log lines. Eg:
* `verify TLS: CA file = '%s'`
* `WARNING: verify TLS: no CA file provided`

Reply:
Yes, that's better. I've updated the PR.